lib/domain/repository/chat.dart

```diff
@@ -42,6 +42,8 @@ abstract class AbstractChatRepository {
   /// Returns reactive map of [RxChat]s in the current pagination view.
   RxObsMap<ChatId, RxChat> get paginated;
 
+  RxObsMap<ChatId, RxChat> get archivedChatsPaginated;
```

Comment:
Why not just `archived` or something? The fact that that field contains chats can be determined by the fact that its type is a map of `RxChat`s.